### ISSUE TITLE
"Something went wrong" message flashes up when creating a repo, and then the repo namespace is wrong

### ISSUE THREAD

**Issue by tom-thorne** [docker/hub-feedback#1025]
I'm using the free tier of docker hub.
Repro:
1. I have no repos
2. I create a private repo called 'test'
3. When it is created, an error banner or page flashes up for literally 1sec, but then the repo details appear and it looks ok.
4. However, instead of a namespace of my-username/test, the namespace is just my-username (verified by inspecting the response to https://hub.docker.com/v2/repositories/my-username/test2/)

When I push an image to that namespace (my-username), it does not end up in the test repo.
Trying to tag and push an image using my-username/test namespace gives an error during push:
"denied: requested access to the resource is denied"

In summary, this means that using the free tier is broken since you can only push one image currently.

**Comment by gibbymt** [comment]
I'm experiencing this as well.

**Comment by rt** [comment]
I have a similar problem.  It flashes "Something went wrong" but creates the correct namespace/repo.  However, it seems corrupt when i try to pull, I get "Error response from daemon: manifest for namespace/repo:latest not found".

I am on Docker for Mac.  Ideas?  Thanks

**Comment by matmeth** [comment]
Me too.

**Comment by hans-permana** [comment]
How did you guys resolve this?

**Comment by niilz** [comment]
I just had the same problem.

I had a CORS PLUGIN ACTIVATED in Chrome that enabled cross-origin sharing (to use fetch requests through localhost from foreign api).

When I DEACTIVATE the PLUGIN the WARNING DISAPPEARS.

**Comment by yancongwen** [comment]
thanks @niilz，your answer help me！

**Comment by liuqiiiiiii** [comment]
I think I'm going to hell

**Comment by innodjet** [comment]
this solve my issue @niilz ..thanks for sharing .

**Comment by simpleprahalad** [comment]
CORS plugin need to be disabled incase you facing the issue of "something went wrong" while creating a new repository. This will solve the problem.

**Comment by securit8** [comment]
i just entered " ; "  end of commit name and it worked. i dont know why but ....

**Comment by singh-sidharth** [comment]
Disable your ad blocker if you are using one.
